### ISSUE TITLE
worker manager is freezed on c-se env

### ISSUE THREAD

**Issue by elbidone** [meumobi/sitebuilder#533]

**Comment by elbidone** [comment]
Fill on operating hand-book steps to resolve issue
https://github.com/meumobi/sitebuilder/wiki/Operating-hand-book

**Comment by danconte72** [comment]
1. Remove All Push Notifications:
```
$ mongo
$ show dbs
$ use [db]
$ show collections
$ db.jobs.find({"type": "push_notification"}).length()
$ db.jobs.find({"type": "push_notification"}).toArray()
$ db.jobs.remove({"type": "push_notification"})
```
2. Find out which Job is frozen
  - a By oldest Job on mongodb:
```
$ db.jobs.find()
{ "_id" : ObjectId("5a28323a701796e3588b457a"), "params" : { "item_id" : "5a28323a701796e3588b4579" }, "created" : ISODate("2017-12-06T18:08:58.032Z"), "modified" : ISODate("2017-12-06T18:08:58.032Z"), "type" : "process_remote_media" }
```
  - b Running perform_works:
```
$ php /sitebuilder/script/perform_works.php
```

3. Check Item by item_id:
```
$ db.items.find({"_id":ObjectId("5a2546e4701796af4c8b4567")})
{ "_id" : ObjectId("5a2546e4701796af4c8b4567"), "address" : "Cielo Auditório - Alameda Xingu, 512 - Alphaville - Barueri/SP", "contact" : "", "created" : ISODate("2017-12-04T13:00:16.869Z"), "description" : "Se inscreva pelo e-mail: cieloday@cielo.com.br", "end_date" : ISODate("2017-12-14T20:00:00Z"), "geo" : NumberLong(0), "groups" : [ ], "guid" : "17746", "is_published" : true, "modified" : ISODate("2018-01-29T16:00:17.225Z"), "notification_id" : "", "order" : NumberLong(207), "parent_id" : NumberLong(236), "published" : ISODate("2017-12-04T13:00:16.869Z"), "site_id" : NumberLong(15), "start_date" : ISODate("2017-12-14T16:00:00Z"), "thumbnails" : [ ], "title" : "Cielo Investors eXperience (Cielo Day - APIMEC)", "type" : "events" }
```

4. Solve the problem :)

5. To make sure, perform step 1 again;

6. Remove all jobs you don't want:
To see all type of jobs:
```db.items.distinct("type")```
To remove:
```db.items.remove({"type":"[type-want-remove]"})```

7. Finally remove the lock:
```$ rm tmp/perform_works.pid```

8. Watch log:
```$ tail -100f log/sitebuilder.log```

**Comment by danconte72** [comment]
[pushs removed](https://github.com/meumobi/sitebuilder/files/1674638/pushs.removed.txt)
